loopdb/sqlc/queries/batch.sql

```diff
@@ -91,22 +91,13 @@ AND
 
 -- name: GetBatchSweeps :many
 SELECT
-        sweeps.*,
-        swaps.*,
-        loopout_swaps.*,
-        htlc_keys.*
+        *
 FROM
         sweeps
-JOIN
```

Comment:
nice, I got it, so you can reuse `GetBatchSweeps` now easier since the loop out data now comes from your interface `LoopOutFetcher.FetchLoopOutSwap`.